pages/dev/send-tokens/interchain-tokens.mdx

```diff
@@ -0,0 +1,58 @@
+import { Callout } from 'nextra-theme-docs'
+
+# Interchain Tokens
+
+Axelar has [several ways](overview) to send [supported tokens](/resources/testnet#assets), but what if you want to send another ERC20 and have it be available on multiple chains? Axelar can help with our new Interchain Tokens!
+
+Our Interchain Token tools register an existing ERC20 (we'll help you create a new one one if you don't have one) and make it available on multiple chains.
```

Comment:
i think we should mention utility here right away: you can send it across chains utilizing bridge front-ends, code into your interchain dApp and send using calls to the gateways, build chain-agnostic wallets on top (that aggregate user's balances across all of their tokens), etc.